src/assets/README.md

```diff
@@ -0,0 +1,15 @@
+# Asset Handling
+
+Fluidic Eleventy uses [Laravel Mix](https://laravel-mix.com) as a wrapper for webpack to process JavaScript and CSS, [Sass](http://sass-lang.com) or [Stylus](http://stylus-lang.com) stylesheets.
+
+## Scripts
+
+By default, the Mix [configuration file](../../webpack.mix.js) will process every JavaScript file that found at the root level of the `/src/assets/scripts` directory. The resulting transpiled  JavaScript files will be written to `/dist/assets/scripts`.
+
+## Stylesheets
+
+The Mix configuration file includes configurations for [CSS](../../webpack.mix.js#L30), [Sass](../../webpack.mix.js#L33) or [Stylus](../../webpack.mix.js#L36). By default, CSS, Sass or Stylus will be run through [Autoprefixer](https://github.com/postcss/autoprefixer). Additional [PostCSS plugins](https://github.com/postcss/postcss/blob/master/docs/plugins.md) may be added ([configuration details can be found here](https://laravel-mix.com/docs/5.0/css-preprocessors#postcss-plugins)).
```

Comment:
It's nice to have direct references but using line numbers here will be brittle, as future changes may affect them, and it may not be obvious for those making code changes that they need to come back here to updated the references. Is there a way to identify where to look for the code without actually having to link to the specific line numbers?

Reply:
@jobara Once this is committed I could link to the file at the specific commit reference.

Reply:
Is there some alternative wording for "configuration details can be found here" to use as the link text. Maybe rewording the sentence like "See [PostCSS Plugins configuration](https://laravel-mix.com/docs/5.0/css-preprocessors#postcss-plugins) for information on how to add and configure additional [plugins](https://github.com/postcss/postcss/blob/master/docs/plugins.md)"

Reply:
Resolved in e97afd9.